ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -39,7 +39,15 @@ import SwiftUI
     @Published public var footnoteTrailingAccessoryView: AnyView?
     @Published public var trailingView: AnyView?
 
-    @objc @Published public var leadingViewSize: MSFListCellLeadingViewSize = .medium
+    @objc @Published public var leadingViewSize: MSFListCellLeadingViewSize = .medium {
+        didSet {
+            if leadingViewSize != oldValue {
+                tokens.cellLeadingViewSize = leadingViewSize
+                tokens.updateForCurrentTheme()
```

Comment:
Why update theme only for specific overrides? How would a client know when there is a need to invoke theme changes manually?

Reply:
This is the way we can implement the change of all the tokens when the style is changed. Similar to the other comment that I responded: the SwiftUI View does not use the enum value, and the tokens class does not subscribe to the @published objects, so we need to force this refresh.

There's definitely room for improvement on this design, but it's beyond the scope of this PR.